test/units/plugins/filter/test_ipaddr.py

```diff
@@ -473,3 +473,32 @@ def test_cidr_merge(self):
         subnets = ['1.12.1.1', '1.12.1.255']
         self.assertEqual(cidr_merge(subnets), ['1.12.1.1/32', '1.12.1.255/32'])
         self.assertEqual(cidr_merge(subnets, 'span'), '1.12.1.0/24')
+
+    def test_ipsubnet(self):
+        address = '1.1.1.1/24'
+        self.assertEqual(ipsubnet(address, '30'), '64')
+        address = '1.1.1.1/25'
+        self.assertEqual(ipsubnet(address, '24'), '0')
+        address = '1.12.1.34/32'
+        subnet = '1.12.1.34/24'
+        self.assertEqual(ipsubnet(address, subnet), '35')
+        address = '192.168.50.0/24'
+        subnet = '192.168.0.0/16'
+        self.assertEqual(ipsubnet(address, subnet), '51')
+        address = '1.12.1.34/24'
+        subnet = '1.12.1.34/32'
+        self.assertEqual(ipsubnet(address, subnet), False)
+        address = '192.168.144.5'
+        subnet = '192.168.0.0/16'
+        self.assertEqual(ipsubnet(address), '192.168.144.5/32')
```

Comment:
Please keep your code style consistent: first you assign vars for some time, then you pass lots of function args directly. There's a better way.